### ISSUE TITLE
External Indices

### ISSUE THREAD

**Issue by mgrundkoetter** [Flowpack/Flowpack.SearchPlugin#66]
We have a project setup with an external Wordpress blog and a NEOS CMS. The WP content is also indexed in the same Elasticsearch by using https://github.com/10up/ElasticPress. The indexed Wordpress content is expected to appear in the search results like the default NEOS content. Is there a way to query a second index with this plugin?

**Comment by dlubitz** [comment]
@mgrundkoetter I'd close this now. Please feel free to re-open if your issue still persist.

**Comment by mgrundkoetter** [comment]
As this was just a yes/no question and I never got an answer, yes I would like to have this reopened. Thanks.

**Comment by dlubitz** [comment]
No, there is no way out of the box. Surely you can implement a second request to the other index and merge somehow the results in fusion or some Eel helper.
